fern/definition/servers/servers/__package__.yml

```diff
@@ -0,0 +1,52 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/fern-api/fern/main/fern.schema.json
+
+imports:
+  commons: ../common.yml
+
+service:
+  auth: true
+  base-path: /servers/servers
+  endpoints:
+    create:
+      path: ""
+      method: POST
+      docs: Create a new dynamic server.
+      request:
+        body: CreateRequest
+      response: CreateResponse
+    destroy:
+      path: "/{server_id}"
+      method: DELETE
+      docs: Destroy a dynamic server.
+      path-parameters:
+        server_id:
+          docs: The id of the server to destroy
+          type: uuid
+      request:
+        body: DestroyRequest
+
+types:
+  CreateRequest:
+    properties:
+      metadata: map<string, unknown>
+      cluster:
+        docs: The name id of the cluster
+        type: string
+      datacenter:
+        docs: The name id of the datacenter
+        type: string
+      # resources:
+      #   cpu:
+      #     docs: The number of CPUs
+      #     type: integer
+      #   memory:
+      #     docs: The amount of memory in MB
+      #     type: integer
+      kill_timeout: integer
+      runtime: commons.Runtime
+  CreateResponse:
+    properties:
+      server: commons.Server
+  DestroyRequest:
+    properties:
+      kill_timeout: optional<integer>
```

Comment:
this might be a string date instead. Change to cleanup milestone, 

```
Yes, ISO 8601 includes a format for durations. It uses the format PnYnMnDTnHnMnS, where P stands for the duration (period), T separates the date and time elements, and the other letters represent years (Y), months (M), days (D), hours (H), minutes (M), and seconds (S). For example, a duration of 3 years, 6 months, 4 days, 12 hours, 30 minutes, and 5 seconds would be P3Y6M4DT12H30M5S.

Would you like a detailed breakdown of how to work with durations in this format in a specific programming context?
```

Reply:
string over number because we don't know units, harder to read